frontend/README.md

```diff
@@ -1,5 +1,13 @@
 # Project created with React + TypeScript + Vite. Uses JSON Server to simulate an external Data Base
 
-1. in project root directory run `npm i` to install all the dependencies
+### Configuration
 
-2. to start the project run `npm run dev` and **ctrl+click** to on the link to open in the browser
+1.  Create a .env file in the frontend folder and define the following variable:
+
+    ```env
+    VITE_PORT=your_desired_port_number
```

Comment:
could u be more precise with the naming of this variable?

Reply:
Good point. I renamed it to more specific VITE_PRODUCTS_DB_PORT